### ISSUE TITLE
Add "--no-delete" option to `reset` to avoid prompting

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction-cli#4]

**Comment by aaronjudd** [comment]
could be the other way as well..  `--delete` could just delete node_modules without prompting.. so that reset worked the same way as meteor.

**Comment by jshimko** [comment]
Overriding the prompt has been implemented in 0.4.1.  You can use the `-y` and `-n` to answer before being asked (similar to `apt-get install -y curl`, for example)

```sh
reaction reset -y
# meteor reset && rm -rf node_modules && meteor npm install 

reaction reset -n
# meteor reset

reaction reset 
# [prompt] Resetting the database! Also delete node_modules? (y/N)
```
